packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -193,8 +201,9 @@ export class AccountForm extends PureComponent {
    * @param  {Object} formState See https://github.com/final-form/final-form#formstate
    * @return {Boolean}
    */
-  isSubmittable({ dirty, initialValues, valid }) {
-    return valid && !(initialValues && !dirty)
+  isSubmittable({ dirty, error, initialValues, valid }) {
+    const untouched = initialValues && !dirty
+    return error || (valid && !untouched)
```

Comment:
So it's submittable when there is an error?

Reply:
Yes, first version was not, but actually, if you have an error somewhere you should be allowed to retry, i.e. to resubmit form.